flocx_market/conf/api.py

```diff
@@ -0,0 +1,23 @@
+from oslo_config import cfg
+
+
+opts = [
+    cfg.HostAddressOpt('host_ip',
+                       default='0.0.0.0'),
+    cfg.PortOpt('port',
+                default=8080),
```

Comment:
8080 is the default keystone port; let's change the default to something OpenStack doesn't use. For reference, here's a list of services and ports that are claimed: https://docs.openstack.org/openstack-helm/latest/devref/endpoints.html